subprojects/execution/src/test/groovy/org/gradle/internal/execution/steps/FingerprinterFixture.groovy

```diff
@@ -40,6 +41,15 @@ abstract trait FingerprinterFixture {
         )
     )
 
+    final OutputFileCollectionFingerprinter outputFingerprinter = new OutputFileCollectionFingerprinter(
```

Comment:
We should probably rename `fingerprinter` to `inputFingerprinter`, too.

Reply:
Done.